Makefile

```diff
@@ -19,8 +22,24 @@ setup: ## Install all the build and lint dependencies
 	dep ensure
 	gometalinter --install
 
+build_all: ## Build a local snapshot binary versions of all commands
+	./scripts/build.sh ${BINARY}
+	make build_geth
```

Comment:
To run make recursively, you should use `$(MAKE)` instead of plain `make` ([see docs](https://www.gnu.org/software/make/manual/html_node/Recursion.html)).

Reply:
But for simple case like this, you should add dependency to target, instead of manually calling make.